docs/docs/configuration/networking-settings.mdx

```diff
@@ -27,6 +28,59 @@ Note that Superset bundles [flask-talisman](https://pypi.org/project/talisman/)
 Self-descried as a small Flask extension that handles setting HTTP headers that can help
 protect against a few common web application security issues.
 
+
+## HTML Embedding of Dashboards and Charts
+
+There are two ways to embed a dashboard: Using the [SDK](https://www.npmjs.com/package/@superset-ui/embedded-sdk) or embedding a direct link. Note that in the latter case everybody who knows the link is able to access the dashboard.
+
+### Enabling Embedding via the SDK
+
+Clicking on `...` next to `EDIT DASHBOARD` on the top right of the dashboard's overview page should yield a drop-down menu including the entry "Embed dashboard".
+
+To enable this entry, add the following line to the `.env` file:
```

Comment:
```suggestion
To enable embedding via the SDK, add the following to your Superset configuration file:
```
I'm thinking this could go in `superset_config.py` and it could be confusing to imply it has to be in the `.env`?

Reply:
I think It is intended as a feature flag. But setting it via the config file doesn't work for some reason. Maybe it's a bug. See #23989